lib/worker.js

```diff
@@ -167,7 +168,20 @@ var HTTPListner = function() {
 				response.end();
 			}
 		},
+		'/analyze-url' : function( request, response ) {
+			response.writeHead( 200, { 'Content-Type': 'text/plain' } );
+			response.write("Analyze");
+			response.end();
```

Comment:
This probably shouldn't write the response until after it adds the job to the queue

Reply:
Makes sense.  This also goes for the code that was already there that handled the request to make a screenshot. I will leave that code as-is for now, there are a lot of things that can be refactored and improved there.